app/src/main/java/ch/epfl/polychef/fragments/PostRecipeFragment.java

```diff
@@ -178,12 +193,14 @@ private boolean buildRecipeAndPostToFirebase() {
             return false;
         } else {
             if(currentMiniature != null) {
-                imageHandler.uploadFromUri(currentMiniature, miniatureName, "TODO:USER", postedRecipe.getUuid().toString());
+                imageHandler.uploadFromUri(currentMiniature, miniatureName, "TODO:USER", postedRecipe.getRecipeUuid());
```

Comment:
Could we link the user to the created recipe already?

Reply:
We add the recipe to the user a few lines below

Reply:
Sorry this is not for the recipe but to link the user to the images of the recipe, but I don't think this is really useful to have in the image's metadata

Reply:
We can remove it later, this PR is already pretty big and it will not cause any problem if it stays that way